wordle-web/src/scripts/wordsService.ts

```diff
@@ -7,9 +7,19 @@ export abstract class WordsService {
     return this.#words.includes(word)
   }
 
-  static validWords(): Array<string> {
+  static validWords(currentWord: string): Array<string> {
     //Todo
-    return new Array<string>()
+
+    const validWordList = new Array<string>()
+
+    for (const word of this.#words) {
+      if (word.startsWith(currentWord) && validWordList.length < 100) {
```

Comment:
Given Grant's comment today, I'm not sure this fully satisfies the requirements. The wording is a little unclear, however.